turbinia/turbiniactl.py

```diff
@@ -717,6 +732,29 @@ def main():
   elif args.command == 'listjobs':
     log.info('Available Jobs:')
     client.list_jobs()
+  elif args.command == 'DumpGCSLogs':
+    if not config.GCS_OUTPUT_PATH:
+      log.error('GCS storage must be enabled in order to use this.')
+      sys.exit(1)
+    if not os.path.isdir(args.output_dir):
+      log.error('Please provide a valid directory path.')
+      sys.exit(1)
+    gcs_file = '**{}**'.format(args.blob_name)
+
+    gcs_bucket = config.GCS_OUTPUT_PATH
+    if args.gcs_path:
+      gcs_bucket = args.gcs_path
+    log.info(
+        'Downloading files from {0!s} to {1!s} using the following query "{2!s}".'
+        .format(gcs_bucket, args.output_dir, gcs_file))
+
+    try:
+      file_path = "{0!s}/{1!s}".format(gcs_bucket, gcs_file)
+      output_writer = GCSOutputWriter(
```

Comment:
Should we keep the directory structure that is in GCS so that we don't have to worry about name conflicts (and file overwrites)?